evo/tools/plot.py

```diff
@@ -183,6 +183,12 @@ def show(self) -> None:
         else:
             plt.show()
 
+    def close(self) -> None:
+        if len(self.figures.keys()) == 0:
```

Comment:
Isn't this check redundant because the for loop will just do nothing if the dict is empty?

Reply:
Indeed, this check is unnecessary. I will remove it.